scripts/grpo_demo_llama3_qwen2.py

```diff
@@ -182,7 +197,7 @@
 else:
   TOTAL_TPU_TO_USE = jax.device_count()
 
-MESH = [(1, TOTAL_TPU_TO_USE), ("fsdp", "tp")]  # YY
+MESH = [(args.rollout_data_parallel_size, TOTAL_TPU_TO_USE // args.rollout_data_parallel_size), ("fsdp", "tp")]
```

Comment:
could you throw a ValueError if TOTAL_TPU_TO_USE % args.rollout_data_parallel_size != 0